sources/Renderer/OpenGL/Shader/GLLegacyShader.cpp

```diff
@@ -75,7 +75,7 @@ std::string GLLegacyShader::GetGLShaderLog(GLuint shader)
         glGetShaderInfoLog(shader, infoLogLength, &charsWritten, infoLog.data());
 
         /* Convert byte buffer to string */
-        return std::string(infoLog.data());
+        return std::string(infoLog.data(), charsWritten);
```

Comment:
Apparently the warning level of all target compilers don't mind this implicit conversation, but I would nonetheless prefer an explicit type cast from `GLsizei` (which is a signed integer) to `std::size_t` here.